tck/src/main/java/ee/jakarta/tck/data/framework/read/only/PositiveIntegers.java

```diff
@@ -42,13 +41,13 @@ public interface PositiveIntegers extends BasicRepository<NaturalNumber, Long> {
 
     boolean existsByIdGreaterThan(Long number);
 
-    KeysetAwarePage<NaturalNumber> findByFloorOfSquareRootNotAndIdLessThanOrderByBitsRequiredDesc(long excludeSqrt,
+    CursoredPage<NaturalNumber> findByFloorOfSquareRootNotAndIdLessThanOrderByBitsRequiredDesc(long excludeSqrt,
                                                                                                   long eclusiveMax,
                                                                                                   PageRequest<NaturalNumber> pagination);
 
     Iterable<NaturalNumber> findByIsOddTrueAndIdLessThanEqualOrderByIdDesc(long max);
 
-    Streamable<NaturalNumber> findByIsOddFalseAndIdBetween(long min, long max);
+    Page<NaturalNumber> findByIsOddFalseAndIdBetween(long min, long max);
```

Comment:
A return type of Page would require a PageRequest parameter.
```suggestion
    List<NaturalNumber> findByIsOddFalseAndIdBetween(long min, long max);
```

Reply:
Yeah sorry, good point.